osde2e/managed_cluster_validating_webhooks_test.go

```diff
@@ -132,27 +150,25 @@ var _ = Describe("Managed Cluster Validating Webhooks", Ordered, func() {
 			},
 		}
 
-		err := client.Create(context.TODO(), pod)
+		err := client.Create(ctx, pod)
+		Expect(err).NotTo(HaveOccurred())
+		err = client.Delete(ctx, pod)
 		Expect(err).NotTo(HaveOccurred())
 	})
 
 	Describe("sre-pod-validation", Ordered, func() {
 		const (
-			privilegedNamespace   = "openshift-backplane"
-			unprivilegedNamespace = "openshift-logging"
-
 			deletePodWaitDuration = 5 * time.Minute
 			createPodWaitDuration = 1 * time.Minute
 		)
 
 		var pod *v1.Pod
 
 		BeforeAll(func() {
-			name := envconf.RandomName("testpod", 12)
 			pod = &v1.Pod{
 				ObjectMeta: metav1.ObjectMeta{
-					Name:      name,
-					Namespace: testNsName,
+					Name:      envconf.RandomName("testpod", 12),
+					Namespace: privilegedNamespace,
```

Comment:
I set privilegedNamespace as the default here because the previous value (testNsName) was always being overridden. privilegedNamespace is used in every current test except for one, and the spec that is not using it can be addressed specific to that test.